docs/en/latest/getting-started.md

```diff
@@ -21,68 +26,109 @@ title: Getting Started
-
-See [Design](./design.md) for more details.
-
-## Installation on Cloud
-
-apisix-ingress-controller supports to be installed on some clouds such as AWS, GCP.
-
-* [Install Ingress APISIX on Azure AKS](https://apisix.apache.org/docs/ingress-controller/deployments/azure)
-* [Install Ingress APISIX on AWS EKS](https://apisix.apache.org/docs/ingress-controller/deployments/aws)
-* [Install Ingress APISIX on ACK](https://apisix.apache.org/docs/ingress-controller/deployments/ack)
-* [Install Ingress APISIX on Google Cloud GKE](https://apisix.apache.org/docs/ingress-controller/deployments/gke)
-* [Install Ingress APISIX on Minikube](https://apisix.apache.org/docs/ingress-controller/deployments/minikube)
-* [Install Ingress APISIX on KubeSphere](https://apisix.apache.org/docs/ingress-controller/deployments/kubesphere)
-* [Install Ingress APISIX on K3S and RKE](https://apisix.apache.org/docs/ingress-controller/deployments/k3s-rke)
+To summarize, APISIX ingress controller has the following features:
 
-## Installation on Prem
+- Declarative configuration with CRDs.
+- Fully dynamic configuration.
+- Supports native Kubernetes Ingress resource (both v1 and v1beta1).
+- Supports service discovery through Kubernetes Service.
+- Out-of-the-box node health check support.
+- Supports load balancing based on pods (Upstream nodes).
+- Rich [Plugins](/docs/apisix/plugins/batch-requests/) with [custom Plugin](/docs/apisix/plugin-develop/) support.
 
-If you want to deploy apisix-ingress-controller on Prem, we recommend you to use [Helm](https://helm.sh/). Just a few steps
+## Get involved
 
-## Get Involved to Contribute
+You can contribute to the development of APISIX ingress controller. See [Development guide](./contribute.md) for instructions on setting up the project locally.
 
-First, your supports and cooperations to make this project better are appreciated.
-But before you start, please read [How to Contribute](./contribute.md)
+See the [Contribute to APISIX](/docs/general/contributor-guide/) section for details on the contributing flow.
 
-## Compatibility with Apache APISIX
+## Compatibility with APISIX
 
-The following table describes the compatibility between apisix-ingress-controller and
-[Apache APISIX](https://apisix.apache.org).
+The table below shows the compatibility between APISIX ingress controller and the APISIX proxy.
 
-| apisix-ingress-controller | Apache APISIX |
-| ----:| ---:|
-| `master` | `>= 2.7`, `2.11` is recommended. |
-| `1.4.0` | `>= 2.7`, `2.11` is recommended. |
-| `1.3.0` | `>= 2.7`, `2.10` is recommended. |
-| `1.2.0` | `>= 2.7`, `2.8` is recommended. |
-| `1.1.0` | `>= 2.7`, `2.7` is recommended. |
-| `1.1.0` | `>= 2.7`, `2.7` is recommended. |
-| `1.0.0` | `>= 2.7`, `2.7` is recommended. |
-| `0.6` | `>= 2.6`, `2.6` is recommended. |
-| `0.5` | `>= 2.4`, `2.5` is recommended. |
-| `0.4` |`>= 2.4`|
+| APISIX ingress controller | Recommended APISIX version |
+| ------------------------: | -------------------------: |
```

Comment:
Hello, after adding `Recommended` in L123, the meaning is changed in my opinion. 🤔 Like in L125, it means `>= 2.7` is recommended.

Reply:
Yes. You are right. I will split this into two columns. I overlooked what is actually mentioned in the column.

Reply:
Updated.